ngraph/core/include/ngraph/opsets/opset.hpp

```diff
@@ -12,99 +12,36 @@
 #include "ngraph/factory.hpp"
 #include "ngraph/ngraph_visibility.hpp"
 #include "ngraph/node.hpp"
+#include "openvino/opsets/opset.hpp"
 
 namespace ngraph {
 /// \brief Run-time opset information
-class NGRAPH_API OpSet {
+class NGRAPH_API OpSet : public ov::OpSet {
```

Comment:
Can you comment why do we have inheritance here? Is the legacy opset usage different from ov::OpSet?

Reply:
Yes, the legacy API has several methods which work with Factory (which we don't want move to new public API).
This inheritance allows to keep methods which use Facroty in old API and hides this Facroty in new API.